client/webserver/site/src/js/wallets.ts

```diff
@@ -647,6 +665,75 @@ export default class WalletsPage extends BasePage {
     window.open(url.toString())
   }
 
+  // displayExportWalletDisclaimer displays a popup which prompts the user to
+  // retype a text confirming that they understand the risks of exporting the
+  // wallet.
+  async displayExportWalletDisclaimer () {
```

Comment:
> prompts the user to retype a text confirming that they understand the risks of exporting the wallet

Mixed feelings about this new security step. It seems a little random. We allow exporting of the application seed, which gives access to all funds from all native wallets, with just a password check. Why would this need stronger security? Or maybe the app seed needs stronger security. I'm also feeling like a password and good messaging is sufficient though.

Reply:
> Why would this need stronger security?

When exporting the funds to an external wallet, there is the additional risk of losing funds when using the wallet at the same time as using the DEX. Most people are aware that if they lose their seed the cannot recover their funds, but they are not aware of this additional risk.

Reply:
No comment yet about the actual text here, but the risk is pronounced with ETH and tokens where:
- ETH needs to be available for txns, including redeems and refunds, which can be time sensitive
- it is very easy to clobber a dex-managed txn by creating txn in the external non-managed wallet that uses the same nonce.  Even if it is just a speed-up, our dex client probably isn't hardened enough to deal with a tx vanishing even if the desired contract interaction is still affected.  The flip side is that DEX can create a txn using a nonce that may have been used by the external wallet clobbering that one.

Reply:
If the external wallet uses a locked UTXO, the trade will be screwed up as well.

Reply:
> We allow exporting of the application seed, which gives access to all funds from all native wallets, with just a password check

@buck54321  You mean by initializing another dexc instance with the app seed or via the `assetseed` CLI app (or advanced/manual coding)?  

> Or maybe the app seed needs stronger security.

Possibly a sentence in the view seed dialog advising not to operate two instances of dexc or the native wallets simultaneously.

> I'm also feeling like a password and good messaging is sufficient though.

No objection to that either I suppose, if it's fairly strong language, even with bold and/or red font for the sentence that says what not to do and what can happen in the worst case scenario.

Reply:
I've updated to using red text, (you can see in the PR description) and removed the extra security step.